kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm.md

```diff
@@ -350,11 +350,11 @@ The `#next [_]` operator initiates execution by:
 -   `#stackDelta` is the delta the stack will have after the opcode executes.
 
 ```k
-    syntax Bool ::= #stackUnderflow ( WordStack , OpCode ) [klabel(#stackUnderflow), macro]
-                  | #stackOverflow  ( WordStack , OpCode ) [klabel(#stackOverflow), macro]
+    syntax Bool ::= #stackUnderflow ( List , OpCode ) [klabel(#stackUnderflow), macro]
+                  | #stackOverflow  ( List , OpCode ) [klabel(#stackOverflow), macro]
  // ---------------------------------------------------------------------------------------
```

Comment:
This is a minimal comment, but as you changed the line below to match the size of the production, what do you think about doing the same here?